2-api/models.md

```diff
@@ -1,68 +1,70 @@
-[Perspective API documentation](../README.md) > [API Reference Docs](README.md) > **Models and attributes**
+[Perspective API documentation](../README.md) > [API Reference Docs](README.md) > **Attributes**
 
-# Models and attributes
+# Attributes
 
-This section details Perspective API’s production and experimental models, which are described in detail in the [All model types](#all-model-types) table below.
+An attribute (sometimes called "models") is a label on which a comment is scored. For example, the some attributes scored by Perspective API include as "Toxcity", "Profanity", "Fliratation", and more. [See all attribute types](#all-attribute-types). Attributes can be in-production or experimental.
```

Comment:
Earlier you wrote that attributes were "previously known as models"; maybe change this here to make this consistent (or change it in the other places to match this)? Or you could say 'sometimes called "model attributes"' which is consistent with other changes in the documentation you've made.